lldb/unittests/Host/PipeTest.cpp

```diff
@@ -55,8 +55,6 @@ TEST_F(PipeTest, OpenAsReader) {
 }
 #endif
 
-// This test is flaky on Windows on Arm.
-#ifndef _WIN32
 TEST_F(PipeTest, WriteWithTimeout) {
```

Comment:
Should we add some tests for the Reader as well?

Something along the lines of:

```
* Read(buf[100], timeout=10s)
* Write("hello world") (under the buffer size)
* AssertEqual(buf, "hello world")

* Read(buf[100], timeout=10s)
* Write("hello world" * 200) (over the buffer size)
* AssertEqual(buf, ("hello world" * 200).slice(0, 100))
* Read(buf[100]) // read the rest
```

Reply:
Good idea. I should have done that originally.